components/sidebar/sidebar.jsx

```diff
@@ -402,7 +400,7 @@ export default class Sidebar extends React.PureComponent {
 
     updateScrollbarOnChannelChange = (channelId) => {
         if (this.refs[channelId]) {
-            const curChannel = this.refs[channelId].getWrappedInstance().refs.channel.getBoundingClientRect();
+            const curChannel = this.refs[channelId].getWrappedInstance().getWrappedInstance().refs.channel.getBoundingClientRect();
```

Comment:
is the second getWrappedInstance() guaranteed to succeed? No need to check for undefined?

Reply:
Yeah, it should. The first one comes from `connect` and the second one comes from `injectIntl`, so they'll always return results